src/components/ReactCompanies/index.tsx

```diff
@@ -8,18 +8,44 @@ import styles from './ReactCompanies.module.scss';
 
 interface ReactCompaniesProps {
   title?: ReactNode;
+  noCompanyMatchInfo?: ReactNode;
 }
 
-const ReactCompanies = ({ title }: ReactCompaniesProps): JSX.Element => {
-  const [selectedTag, setTag] = useState<string>(ALL_TAG);
+const ReactCompanies = ({ title, noCompanyMatchInfo }: ReactCompaniesProps): JSX.Element => {
```

Comment:
Czy ten komponent będzie jeszcze gdzieś używany, że potrzebuje przyjmować propsy? Jeśli nie, to wystarczy elementy w nich przekazywane umieścić w jego JSX-ie

Reply:
Ponieważ komponenty przekazane w propsach są komponentami Astro to, żeby mogły być wyrenderowane w komponencie Reactowym to musza być przekazane w ten sposób 😇